ansible/tasks/initdb.yml

```diff
@@ -2,95 +2,32 @@
 # This task will initialize the immortal DBs in the database account.
 # This step is usually done only once per account.
```

Comment:
`  # This step is usually done only once per account. ` per account?  how about per deployment.